pkg/security/resolvers/process/resolver_ebpfless.go

```diff
@@ -101,7 +101,7 @@ func (p *EBPFLessResolver) AddExecEntry(key CacheResolverKey, file string, argv
 
 	entry.Process.ArgsEntry = &model.ArgsEntry{Values: argv}
 	if len(argv) > 0 {
-		entry.Process.Comm = argv[0]
+		entry.Process.Comm = filepath.Base(file)
```

Comment:
I'm not sure about that change. We need to check why you had to change it

Reply:
It was a needed change to make the test `TestProcessContext/self-exec-std` pass. It uses selftest to self exec (using /proc/self/exe). It could be updated later if we add a proc fs fallback to catch the "real" comm. Since then, I think this change is ok